src/lustre/lustreAstNormalizer.ml

```diff
@@ -779,15 +769,15 @@ and normalize_node info map
       info, empty ()
   in
   (* Normalize equations and the contract *)
-  let nitems, gids4, warnings2 = normalize_list (normalize_item info map) items in
+  let nitems, gids4, warnings2 = normalize_list (normalize_item node_id info map) items in
```

Comment:
Please check out whether passing `node_id` is really necessary.

Reply:
Addressed. There was one instance where it was being used (collecting some context information within normalize_node), but all other references were unnecessary.